content/chapter-03/SystemMetricsLab.md

```diff
@@ -1 +1,40 @@
-# Lab: common tools for system metrics
\ No newline at end of file
+# Lab: common tools for system metrics
+
+* In this lab, two Virtual Machine instances will be created: One will run a stress test, and the other will run the application to be tested.
+
+* Tester instance
+  * Create a small instance to launch a stress test tool. e.g.: AWS EC2 t2.micro, AMI: Ubuntu Server 16.04 LTS (HVM), SSD Volume Type - ami-66506c1c
```

Comment:
Create a small instance to launch the stress test tool, e.g.: AWS EC2 t2.micro with Ubuntu Server 16.04 LTS

Reply:
Done